website/docs/r/iam_policy.html.markdown

```diff
@@ -28,7 +28,6 @@ resource "ovh_iam_policy" "manager" {
     "account:apiovh:me/certificates/get",
     "account:apiovh:me/tag/get",
     "account:apiovh:services/get",
-    "account:apiovh:*",
```

Comment:
I found it was a good thing to have at least one example with a wildcard.

That illustrated that this field could take several values.

Thus I would keep at least a mention of wildcard (and thus either change the policy name or add a second one in example)